src/components/InputText/InputText.tsx

```diff
@@ -18,6 +18,10 @@ export interface InputTextProps extends InputHTMLAttributes<HTMLInputElement> {
   helpText?: string;
   errorText?: string;
   requiredText?: string;
+  /**
```

Comment:
this is only for the docs site demo, right? maybe we can note that it's for demo purposes and add a TODO to remove it and use the storybook features?

Reply:
Could there be a use case to force validation in a product? e.g. loading a page with a form on it that already has data entered?